src/sentry/api/event_search.py

```diff
@@ -1166,10 +1168,15 @@ def generic_visit(self, node, children):
     },
 )
 
+QueryOp = Literal["AND", "OR"]
+QueryToken = Union[SearchFilter, QueryOp, ParenExpression]
```

Comment:
Moved the types here, so that we can slowly properly use them across the board, since `list[SearchFilter]` is the wrong type return by `parse_search_query`.